docs/proposals/llm-hyperparameters-tuning-api.md

```diff
@@ -0,0 +1,135 @@
+# LLM Hyperparameters Tuning API Proposal
+
+- [LLM Hyperparameters Tuning API Proposal](#llm-hyperparameters-tuning-api-proposal)
+  * [Links](#links)
+  * [Motivation](#motivation)
+  * [Goal](#goal)
+  * [Design for API](#design-for-api)
+    + [Example](#example)
+  * [Implementation](#implementation)
+  * [Advanced Functionalities](#advanced-functionalities)
+
+## Links
+
+- [katib/issues#2291 (Tuning API in Katib for LLMs)](https://github.com/kubeflow/katib/issues/2291)
+
+## Motivation
+
+LLMs are experiencing rapid growth in popularity and evolution, leading to an increased demand for LLMOps in Kubernetes environment. Hyperparameter tuning can greatly improve model performance but is often labor-intensive for humans. We want to provide an LLM hyperparameters tuning API which automates the process and is user-friendly for data scientists.
```

Comment:
It would be nice to explain background to what we did for Training Operator: https://www.kubeflow.org/docs/components/training/user-guides/fine-tuning/

Reply:
I have added this background. Please check the latest version.